src/Maker/MakeAuthenticator.php

```diff
@@ -194,20 +194,43 @@ function ($answer) {
                     true
                 )
             );
+
+            $command->addArgument('support-remember-me', InputArgument::REQUIRED);
+            $input->setArgument(
+                'support-remember-me',
+                $io->confirm(
+                    'Do you want to support remember me?',
+                    true
+                )
+            );
+
+            if($input->getArgument('support-remember-me')) {
+                $command->addArgument('always-remember-me', InputArgument::REQUIRED);
+                $input->setArgument(
+                    'always-remember-me',
+                    $io->confirm(
+                        'Do you want remember me to be activated via a checkbox or always activated?',
+                        false
+                    )
```

Comment:
Hmm, the question isn't very clear. Perhaps the user should choose between 2 options instead of a yes/no:

[0] Activate when the user checks a box
[1] Always activate remember me